docs/API.md

```diff
@@ -668,6 +669,70 @@ except ResponseError as err:
     print(err)
 ```
 
+<a name="select_object_content"></a>
+### select_object_content(self, bucket_name, object_name, obj)
+Select object content filters the contents of object based on a simple structured query language (SQL).
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|:---|:---|:---|
+|``bucket_name``   |_string_   |Name of the bucket.   |
+|``object_name``   |_string_   |Name of the object.   |
+|``obj`` | _Object_ | Query Options   |
+
+
+__Return Value__
+
+|Param   |Type   |Description   |
+|:---|:---|:---|
+|``object``   | _urllib3.response.HTTPResponse_   |Represents http streaming reader.   |
+
+__Example__
+
+
+```py
+client = Minio('s3.amazonaws.com',
+               access_key='YOUR-ACCESSKEY',
+               secret_key='YOUR-SECRETKEY')
+
+obj = SelectObjectOptions(
+    expression="select * from s3object",
+    input_serialization = InputSerialization(compression_type="NONE",
+                              csv=CSVInput(FileHeaderInfo="USE",
+                                           RecordDelimiter="\n",
+                                           FieldDelimiter=",",
+                                           QuoteCharacter='"',
+                                           QuoteEscapeCharacter='"',
+                                           Comments="#",
+                                           AllowQuotedRecordDelimiter="FALSE"
+                                           ),
+                                ),
+    output_serialization=OutputSerialization(
+                    csv = CSVOutput(
+                        QuoteFields="ASNEEDED",
+                        RecordDelimiter="\n",
+                        FieldDelimiter=",",
+                        QuoteCharacter='"',
+                        QuoteEscapeCharacter='"',
+                        )
+                    ),
+    request_progress=RequestProgress(
+        enabled="TRUE"
+        )
+    )
+
+
+try:
+    records, stats = client.select_object_content('sinha', 'test.csv', obj)
```

Comment:
This example should be updated

Reply:
done

Reply:
@sinhaashish we can change the bucket & object names here